xchange-stream-service-netty/src/main/java/info/bitrich/xchangestream/service/netty/NettyStreamingService.java

```diff
@@ -163,8 +163,7 @@ protected Completable openConnection() {
                 if (ssl) {
                   SslContextBuilder sslContextBuilder = SslContextBuilder.forClient();
                   if (acceptAllCertificates) {
-                    sslContextBuilder =
-                        sslContextBuilder.trustManager(InsecureTrustManagerFactory.INSTANCE);
+                    sslContextBuilder.trustManager(InsecureTrustManagerFactory.INSTANCE);
```

Comment:
Looks like the fmt plugin needs rerunning here

Reply:
reran (nothing changed here but gemini streaming service did)